docker-compose.yml

```diff
@@ -21,8 +21,8 @@ services:
     profiles: [production, test, default]
     image: 'memcached:1.6'
     restart: unless-stopped
-    # cache memory size - 4GB
-    command: -m 4096
+    # cache memory size - 4GB, maximum item size - 5MB
```

Comment:
Wondering if bumping up to 4/5MB is enough? Any reason not to go to say 8MB, since we return SVGs and such from some preprocessors?